### PR TITLE
Release/2.1.1 => master

### PR DESCRIPTION
**Pull request by latture**
New release to incorporate the fix for errors thrown when trying to encode numpy values.

### CODE REVIEW

File: pypif/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '1.2.0'
+__version__ = '2.1.0'
```
Comment (maxhutch) [inline pypif/__init__.py]: Can you bump this @latture ?

File: pypif/tests/test_pif.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
-from pypif.pif import loads, dumps
-from pypif.obj.system import System
+import numpy as np
```
Comment (maxhutch) [inline pypif/tests/test_pif.py:1]: Numpy isn't listed in the requirements or setup.py

**Review by maxhutch — DISMISSED** [review]

File: pypif/tests/test_pif.py (hunk 2 above)
Comment (latture) [reply pypif/tests/test_pif.py:1]: Thanks, I should've caught that.

File: pypif/util/pif_encoder.py [hunk 3]
```diff
@@ -18,5 +20,11 @@ def default(self, obj):
...
         elif isinstance(obj, list):
             return [i.as_dictionary() for i in obj]
-        else:
+        elif hasattr(obj, 'as_dictionary'):
             return obj.as_dictionary()
+        elif isinstance(obj, (np.int_, np.intc, np.intp, np.int8, np.int16, np.int32,
+                              np.int64, np.uint8, np.uint16, np.uint32, np.uint64)):
+            return int(obj)
+        elif isinstance(obj, (np.float_, np.float16, np.float32, np.float64)):
+            return float(obj)
```
Comment (sparadiso) [inline pypif/util/pif_encoder.py:29]: Why not just cast to float? Is there a reason to maintain the int/float distinction during serialization?

**Review by sparadiso — APPROVED** [review]
lgtm

**Comment by latture** [comment]
@maxhutch Is the latest commit with changes to `requirements.txt` consistent with your expectation for the `setup.py <-> requirements.txt` parity?

**Review by maxhutch — APPROVED** [review]

**Pull request merged**